update.sh

```diff
@@ -12,17 +12,29 @@ versions=( "${versions[@]%/}" )
 travisEnv=
 for version in "${versions[@]}"; do
 	travisEnv='\n  - VERSION='"$version$travisEnv"
-	
-	repoBase="http://packages.elasticsearch.org/elasticsearch/$version/debian"
-	if [ "${version%%.*}" -ge 2 ]; then
-		repoBase="http://packages.elasticsearch.org/elasticsearch/${version%%.*}.x/debian"
+
+	rcVersion="${version%-rc}"
```

Comment:
Also, when it comes down to it, we only care whether it's a pre-release or not since we don't usually want to add `latest` to a pre-release. :smile: